scripts/analytics/node_log_events.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from modularodm import Q
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from website.app import init_app
+from website.project.model import NodeLog
+from scripts.analytics.base import EventAnalytics
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class NodeLogEvents(EventAnalytics):
+
+    @property
+    def collection_name(self):
+        return 'node_log_events'
+
+    def get_events(self, date):
+        """ Get all node logs from a given date for a 24 hour period,
+        ending at the date given.
+        """
+        super(NodeLogEvents, self).get_events(date)
+
+        # In the end, turn the date back into a datetime at midnight for queries
+        date = datetime(date.year, date.month, date.day)
```

Comment:
`datetime` build naive datetimes by default.  This might start erroring when we move to django-osf, so it would be good to explicitly pass the utc timezone here.